### ISSUE TITLE
Gives 401 on a public resource if fed expired tokens

### ISSUE THREAD

**Issue by timbl** [solid/node-solid-server#1176]
## Please describe what you did in reproducible steps

Use an old bowser window
Load a public 

## How did it work with 4.x series servers?

200 works

## Any material that will help, logs, error messages, etc.

Server logs:

```
  solid:ACL accessDenied: modeURIorReasons: ["User Unauthorized","http://www.w3.org/ns/auth/acl#Read"] +0ms
  solid:ACL  checking <http://www.w3.org/ns/auth/acl#Write> +0ms
  solid:ACL   MODE REQUIRED NOT ALLOWED: <http://www.w3.org/ns/auth/acl#Write> Denying with User Unauthorized +0ms
  solid:ACL could not fetch owner doc NamedNode { termType: 'NamedNode', value: 'https://timbl.com/{{webId}}' } { [HTTPError: File not found: /Library/WebServer/Documents/{{webId}}]
  name: 'HTTPError',
  message: 'File not found: /Library/WebServer/Documents/{{webId}}',
  status: 404 } +0ms
  solid:ACL accessDenied: checking access to <https://timbl.com/favicon.ico> by null and origin null +1ms
  solid:ACL    2 direct authentications about <https://timbl.com/favicon.ico> +0ms
  solid:ACL    Checking auth <https://timbl.com/favicon.ico.acl#owner> with agent null +0ms
  solid:ACL     Agent or group: Fail: not public and not logged on. +0ms
  solid:ACL      The agent/group check fails +0ms
  solid:ACL       Check failed: User Unauthorized +0ms
  solid:ACL     Agent or group: Ok, its public. +0ms
  solid:ACL       Mode allowed: <http://www.w3.org/ns/auth/acl#Read> +0ms

```

**Comment by timbl** [comment]
Some more server log for a different resource
```
  solid:server Error page because of: { Error: Unauthorized
    at AuthenticatedRequest.unauthorized (/devel/github.com/solid/node-solid-server/node_modules/@solid/oidc-rs/src/AuthenticatedRequest.js:750:17)
    at validateExpiry (/devel/github.com/solid/node-solid-server/node_modules/@solid/oidc-rs/src/AuthenticatedRequest.js:616:22)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
  handled: true,
  statusCode: 401,
  realm: 'https://timbl.com',
  error: 'invalid_token',
  error_description: 'Access token is expired',
  error_uri: undefined } +76ms
  solid:server Display login-required for https://timbl.com/timbl/Public/Archive/ +2ms
  solid:index Looking for index in /timbl/Public/Archive/ +151ms
  solid:ACL Using ACL https://timbl.com/timbl/Public/Archive/.acl for ./ +5ms
  solid:ACL    3 direct authentications about <https://timbl.com/timbl/Public/Archive/> +1ms
  solid:ACL accessDenied: checking access to <https://timbl.com/timbl/Public/Archive/> by null and origin null +156ms
  solid:ACL    3 direct authentications about <https://timbl.com/timbl/Public/Archive/> +0ms
  solid:ACL    Checking auth <https://timbl.com/timbl/Public/Archive/.acl#ControlReadWrite> with agent null +0ms
  solid:ACL     Agent or group: Fail: not public and not logged on. +1ms
  solid:ACL      The agent/group check fails +0ms
  solid:ACL       Check failed: User Unauthorized +0ms
  solid:ACL     Agent or group: Ok, its public. +0ms
  solid:ACL       Mode allowed: <http://www.w3.org/ns/auth/acl#Read> +0ms
  solid:ACL    Checking auth <https://timbl.com/timbl/Public/Archive/.acl#ReadWrite> with agent null +0ms
  solid:ACL     Agent or group: Fail: not public and not logged on. +0ms
  solid:ACL      The agent/group check fails +0ms
  solid:ACL       Check failed: User Unauthorized +0ms
  solid:ACL accessDenied: modeURIorReasons: ["User Unauthorized","http://www.w3.org/ns/auth/acl#Read"] +0ms
  solid:ACL  checking <http://www.w3.org/ns/auth/acl#Read> +0ms
  solid:ACL   Mode required and allowed:<http://www.w3.org/ns/auth/acl#Read> +0ms
  solid:ACL Using ACL https://timbl.com/timbl/Public/Archive/.acl for ./ +0ms
  solid:ACL Using ACL https://timbl.com/timbl/Public/Archive/.acl for ./ +0ms
  solid:ACL Using ACL https://timbl.com/timbl/Public/Archive/.acl for ./ +1ms
  solid:ACL Using ACL https://timbl.com/timbl/Public/Archive/.acl for ./ +0ms
  solid:ACL Using ACL https://timbl.com/timbl/Public/Archive/.acl for ./ +0ms
  solid:ACL Using ACL https://timbl.com/timbl/Public/Archive/.acl for ./ +0ms
  solid:ACL Using ACL https://timbl.com/timbl/Public/Archive/.acl for ./ +0ms
  solid:ACL    3 direct authentications about <https://timbl.com/timbl/Public/Archive/> +0ms
  solid:ACL    3 direct authentications about <https://timbl.com/timbl/Public/Archive/> +2ms
  solid:ACL    3 direct authentications about <https://timbl.com/timbl/Public/Archive/> +1ms
  solid:ACL    3 direct authentications about <https://timbl.com/timbl/Public/Archive/> +1ms
  solid:ACL    3 direct authentications about <https://timbl.com/timbl/Public/Archive/> +2ms
  solid:ACL    3 direct authentications about <https://timbl.com/timbl/Public/Archive/> +1ms
  solid:ACL    3 direct authentications about <https://timbl.com/timbl/Public/Archive/> +2ms
  solid:ACL accessDenied: checking access to <https://timbl.com/timbl/Public/Archive/> by null and origin null +179ms
  solid:ACL    3 direct authentications about <https://timbl.com/timbl/Public/Archive/> +0ms
  solid:ACL    Checking auth <https://timbl.com/timbl/Public/Archive/.acl#ControlReadWrite> with agent null +0ms
  solid:ACL     Agent or group: Fail: not public and not logged on. +0ms
  solid:ACL      The agent/group check fails +0ms
  solid:ACL       Check failed: User Unauthorized +0ms
  solid:ACL     Agent or group: Ok, its public. +0ms
  solid:ACL       Mode allowed: <http://www.w3.org/ns/auth/acl#Read> +0ms
  solid:ACL    Checking auth <https://timbl.com/timbl/Public/Archive/.acl#ReadWrite> with agent null +0ms
  solid:ACL     Agent or group: Fail: not public and not logged on. +1ms
  solid:ACL      The agent/group check fails +0ms
  solid:ACL       Check failed: User Unauthorized +0ms
  solid:ACL accessDenied: modeURIorReasons: ["User Unauthorized","http://www.w3.org/ns/auth/acl#Read"] +0ms
  solid:ACL  checking <http://www.w3.org/ns/auth/acl#Append> +0ms
  solid:ACL   MODE REQUIRED NOT ALLOWED: <http://www.w3.org/ns/auth/acl#Append> Denying with User Unauthorized +0ms
```

**Comment by jaxoncreed** [comment]
Fixed in #1177
